AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -239,7 +239,12 @@ - (void)startUpdate {
   NSString *releaseHash = MSPackageHash();
   if (releaseHash) {
     [self changeDistributionGroupIdAfterAppUpdateIfNeeded:releaseHash];
-    NSString *updateToken = [MSKeychainUtil stringForKey:kMSUpdateTokenKey];
+    OSStatus statusCode;
+    NSString *updateToken = [MSKeychainUtil stringForKey:kMSUpdateTokenKey statusCode:&statusCode];
+    if (statusCode == errSecInteractionNotAllowed) {
```

Comment:
Shouldn't you log errors for all failed status codes?

Reply:
Keychain util will log.

Reply:
The logic here is more about returning than logging.